localizations/es.json

```diff
@@ -0,0 +1,65 @@
+{
+	"Language":"Español",
+	"Payoff":"Simplemente un motor wiki de archivos planos de markdown sin base de datos.",
+	"Subtitle":"Motor wiki de archivos planos de markdown",
+	"TooltipSidebar":"Barra lateral",
+	"TooltipPrint":"Imprimir este documento",
+	"TooltipSignIn":"Iniciar sesión para editar o<br>agregar nuevos documentos",
+	"TooltipSettings":"Configuración",
+	"TooltipNewDocument":"Agregar nuevo documento",
+	"TooltipEditDocument":"Editar documento",
+	"TooltipCancelEditing":"Cancelar edición",
+	"TooltipImages":"Imágenes",
+	"TooltipDeleteDocument":"Eliminar este contenido",
+	"TooltipDeleteDocumentConfirm":"¿Realmente desea eliminar este contenido?",
+	"TooltipVersioning":"Realizar copia de seguridad de la versión actual",
+	"TooltipSave":"Guardar",
+	"Search":"Buscar en la wiki.",
+	"SearchResults":"Resultados de búsqueda",
+	"SearchNoResults":"No se encontraron resultados para",
+	"Logout":"Cerrar sesión",
+	"LastUpdate":"Esta página fue editada por última vez el",
+	"PoweredBy":"Desarrollado por",
+	"CookieAgreement":"Acuerdo de cookies",
+	"CookieButtonDisagree":"RECHAZAR",
+	"CookieButtonAgree":"ACEPTAR",
+	"Images":"Imágenes",
+	"ImagesBrowse":"Explorar",
+	"ImagesSelect":"Seleccionar una imagen para cargar",
+	"ImagesSubmit":"Cargar",
```

Comment:
"Attachments": "Attachments",
	"AttachmentsBrowse": "Browse",
	"AttachmentsSelect": "Select a file to upload",
	"AttachmentsSubmit": "Upload",